collect_app/src/main/java/org/odk/collect/android/logic/PropertyManager.java

```diff
@@ -94,7 +94,7 @@ public PropertyManager reload() {
             putProperty(PROPMGR_DEVICE_ID,     "",         deviceDetailsProvider.getDeviceId());
             putProperty(PROPMGR_PHONE_NUMBER,  SCHEME_TEL,          deviceDetailsProvider.getLine1Number());
         } catch (SecurityException e) {
-            Timber.e(e);
+            Timber.i(e);
```

Comment:
This is the top reporting issue in Firebase but it's expected because there could be an attempt to get the phone number before permissions were prompted for.